pennylane_cirq/simulator_device.py

```diff
@@ -190,14 +190,6 @@ def generate_samples(self):
             [self._result.measurements[str(wire)].flatten() for wire in range(self.num_wires)]
         ).T.astype(int)
 
-    def expval(self, observable):
-        # pylint: disable=missing-function-docstring
-        if not hasattr(self._simulator, "simulate_expectation_values"):
-            return super().expval(observable)
-        return self._simulator.simulate_expectation_values(
-            self.circuit, cirq.PauliSum() + self.to_paulistring(observable)
-        )[0]
```

Comment:
Would we want to keep this, but simply support the new `simulate_expectation_values` API?

It would be easier to support this though onces Shots II has been merged

Reply:
Agreed. I think I'll add that in post shots 2 refactor. This code was only for Qhack/Floq (and based on the user feedback, I was wrong in ignoring the shots value here).